### ISSUE TITLE
Dockerise Deploys

### ISSUE THREAD

**Issue by sansyrox** [mexili/incogly#6]
Create Docker containers to dockerise deploys

**Comment by subhangi2731** [comment]
can you please assign this issue to me? @sansyrox

**Comment by subhangi2731** [comment]
can you please assign this issue to me? @sansyrox

**Comment by sansyrox** [comment]
Yes @subhangi2731 , go ahead!

**Comment by paramsiddharth** [comment]
I would like to work on this! May I be assigned?

**Comment by sansyrox** [comment]
Yes, @paramsiddharth sure. We have the docker file in the repo. Can you couple it with nginx and docker compose?

**Comment by paramsiddharth** [comment]
> Yes, @paramsiddharth sure. We have the docker file in the repo. Can you couple it with nginx and docker compose?

Although I know about creating the `Dockerfile` and setting up `nginx`, I don't know how to create the `docker-compose.yml` yet. I'll study it and get back to you.
